addons/mail/static/src/views/web/fields/activity_exception/activity_exception.js

```diff
@@ -30,5 +29,5 @@ Object.assign(ActivityException, {
 registry.category("fields").add("activity_exception", {
     component: ActivityException,
     fieldDependencies: ActivityException.fieldDependencies,
-    noLabel: true,
+    label: false,
```

Comment:
this change should be explained in the commit message (which should be a bit more detailed anyway)